external-crates/move/crates/move-compiler/tests/move_2024/typing/macros_lambdas_checked_invalid_arity.exp

```diff
@@ -66,15 +90,15 @@ error[E04010]: cannot infer type
    │                    ^^^^^^^^ Could not infer this type. Try adding an annotation
 
 error[E04007]: incompatible types
-   ┌─ tests/move_2024/typing/macros_lambdas_checked_invalid_arity.move:20:17
+   ┌─ tests/move_2024/typing/macros_lambdas_checked_invalid_arity.move:19:19
    │
 19 │         foo!<u64, vector<u8>>(
-   │                   ---------- Expected: 'vector<u8>'
+   │                   ^^^^^^^^^^
+   │                   │
+   │                   Invalid type annotation
+   │                   Expected: 'vector<u8>'
 20 │             |_| (vector<u8>[], vector<u8>[]), // invalid
-   │                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-   │                 │
-   │                 Invalid type annotation
-   │                 Given: '(vector<u8>, vector<u8>)'
+   │                 ---------------------------- Given: '(vector<u8>, vector<u8>)'
```

Comment:
These errors swapped their primary and secondary locations. Would it be possible to swap them back (to better-preserve error ordering)?

Reply:
I could try... maybe. I think I'd have to join first then subtype the joined result

Reply:
I tried. It did not work. This one in particular is a bit fucked in that it is the result of the lambda body getting a `_` annotation now, so it only fails once it gets to the return type